src/Config/DefaultConfig.php

```diff
@@ -2,12 +2,12 @@
 
 namespace Acquia\Drupal\RecommendedSettings\Config;
 
-use Consolidation\Config\Config;
+#use Consolidation\Config\Config;
```

Comment:
If this is not required can we remote it.

Reply:
done